lib/common.js

```diff
@@ -1,10 +1,39 @@
 const path = require('path')
 const compression = require('compression')
 const express = require('express')
+const fetch = require('node-fetch')
+const fs = require('fs')
 
 function setupRoutes (app, prefix = '') {
   app.use(compression())
   app.use(prefix + '/', express.static(path.join(__dirname, '../public')))
+  app.get('/getSkin', (req, res) => {
+    if (!req.query.name) return res.status(400).send('Missing url')
+    const name = req.query.name
+
+    fetch('https://api.ashcon.app/mojang/v2/user/' + name)
```

Comment:
Why use a 3rd party api instead of the mojang one ? https://sessionserver.mojang.com/session/minecraft/profile/[uuid]

Reply:
I started using this one cause a friend of mine recommended it to me, but if you want I can try to use the mojang apis instead

Reply:
mojang API will be more reliable and will look identical to how Minecraft client renders the skins, without any caching from 3rd party sites

```js
let uuid = await fetch(`https://api.mojang.com/users/profiles/minecraft/${username}?at=Date.now()`).then(r => r.json()).then(r => r.id)
let r = await fetch(`https://sessionserver.mojang.com/session/minecraft/profile/${uuid}`).then(r => r.json())
let skinData = JSON.parse(Buffer.from(r.properties[0].value, 'base64').toString('utf8'))
let url = skinData.textures.SKIN.url
```